repository/c.json

```diff
@@ -3807,7 +3818,6 @@
 		{
 			"name": "CopyToSharePointRoot",
 			"details": "https://github.com/joladev/SublimeCopyToSharePointRoot",
-			"author": "eakron",
```

Comment:
This looks like an unintentional change?

Reply:
Whoops, fixed!